site/roles/trinity/ssl-cert/tasks/main.yml

```diff
@@ -1,16 +1,21 @@
 ---
-- name: Ensure '{{ ssl_cert_owner }}' user exists
-  user:
-    name: '{{ ssl_cert_owner }}'
-    uid: '{{ ssl_cert_owner_id }}'
-    system: yes
-  when: ssl_cert_owner not in 'root'
+- block:
+
+  - name: Ensure '{{ ssl_cert_owner }}' user exists
+    user:
+      name: '{{ ssl_cert_owner }}'
+      uid: '{{ ssl_cert_owner_id }}'
+      system: yes
+    when: ssl_cert_owner not in 'root'
+  
+  - name: Ensure '{{ ssl_cert_group }}' group exists
+    group:
+      name: '{{ ssl_cert_group }}'
+      gid: '{{ ssl_cert_group_id }}'
+      system: yes
 
-- name: Ensure '{{ ssl_cert_group }}' group exists
-  group:
-    name: '{{ ssl_cert_group }}'
-    gid: '{{ ssl_cert_group_id }}'
-    system: yes
+  when: ansible_connection not in 'lchroot'
```

Comment:
maybe replace the double condition with something like `on_controller` set in the `init` role